app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,18 @@
+from .elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+        self, nickname: str, musical_instrument: str, bow_level: int
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (
+            f"Elf ranger {self.nickname}. "
+            f"{self.nickname} has bow of the {self._bow_level} level"
+        )
+
+    def get_rating(self) -> None:
+        return 3 * self._bow_level
```

Comment:
The `get_rating` method is annotated to return `None`, but it actually returns an `int`. You should change the return type annotation to `int` to match the actual return type.